### ISSUE TITLE
AttributeError: module 'keras.backend' has no attribute 'backend'

### ISSUE THREAD

**Issue by micklexqg** [fchollet/keras#6602]
I installed keras2.0.0 and tensorflow1.1.0, my python is 3.5.2(within anaconda)
but when I run the flowing code:
from keras import backend as K

got the error:
AttributeError: module 'keras.backend' has no attribute 'backend'
and i find there is 'backend' in the keras.json.

How to resolve this?

**Comment by Clemence-ECL** [comment]
If you ever copied/pasted the code from github (for instance the one from the fchollet) it won't work, try to rewrite it.

**Comment by ardianumam** [comment]
Why does "copied/pasted the code from github" give effect like that?

**Comment by Clemence-ECL** [comment]
@ardianumam No idea really. I changed it and it worked. But I might be wrong. Maybe changing the name of the file is enough.

**Comment by ardianumam** [comment]
@Clemence-ECL : Yes, because my code (code "x") could be run well both using tensorflow backend and theano backend before. But, after I tried a code from fchollet by doing copy-paste in a new python file in the same project folder, when I run again code "x" using tensorflow, it gave error like that (module 'keras.backend' has no attribute 'backend'). But, when the backend is using theano, code "x" runs well. I just tried to rename the file and still got error. Don't know why.
Here is my specs : I use Python 3.6 64-bit on Windows 10, Keras 2.0.2, TensorFlow1.2.0, Theano 0.9.0
cc : @micklexqg
